src/java/org/apache/cassandra/db/compaction/CompactionTask.java

```diff
@@ -128,6 +128,10 @@ protected void runMayThrow() throws Exception
 
         final long expectedWriteSize = checkAvailableDiskSpaceAndGetWriteSize(checkAvailableDiskSpaceFunction);
 
+        if (expectedWriteSize > 5 * 1024 * 1024 * 1024) {
+            logger.info(String.format("Compaction for ks/cf %s/%s exceeds 5GiB with total size of %d", cfs.keyspace, cfs.name, expectedWriteSize));
```

Comment:
we want to make the logs safe loggable, something like
```suggestion
            logger.info("Compaction for ks/cf {}/{} exceeds 5GiB with total size of {}", SafeArg.of("keyspace", cfs.keyspace), SafeArg.of("columnFamily", cfs.name), SafeArg.of("size", expectedWriteSize)));
```

Reply:
also please match the formatting style of cassandra, so `{` on new lines